projects/step-frontend/src/lib/modules/_common/components/sidebar/sidebar.component.ts

```diff
@@ -55,7 +67,7 @@ export class SidebarComponent implements AfterViewInit, OnDestroy {
 
   ngAfterViewInit(): void {
     this._sideBarState.initialize();
-    this._menuItems$.subscribe(() => {
+    this.displayMenuItems$.subscribe(() => {
```

Comment:
This logic is relates to sidebar initialization, that should happened once, as soon as menu items will be retrieved (from backend in case of EE, or from menu registry due the initialization).
Subscribing to `displayMenuItems$` will re-invoke it each time when bookmark list will be changed.
I think that it should continue to listen `_menuItems$` instead of `displayMenuItems$`

Reply:
Changed it back to _menuItems$